### PR TITLE
Clean up the Task

### PR DESCRIPTION
**Pull request by adamwitko-s4rb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
+- Pushed to a Github repository
+- Shared with your Technical contact at S4RB
+- If you're invited for a chat with S4RB we'll aim to talk about your submission. Be ready for questions on your solution!
+
+## 1. Serve up the [complaints](/data/complaints.csv) data from an API endpoint
+
+### Requirements
+
+- Use Express, Hapi, vanilla HTTP, or any other server-side web framework
+- Requests should be handled to responed with a CSV file
```
Comment (MattHodges-S4RB) [inline README.md]: respond

File: README.md [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
+- Requests should be handled to responed with a JSON file
```
Comment (MattHodges-S4RB) [inline README.md]: and again

File: README.md [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
+
+You will be extending on this to create the online portal!
+
+## 2. Extend your endpoint to include the Complaints per million units (CPMU) based on the [complaints](/data/complaints.csv) data
+
+- The file [cpmu.xlsx](/cpmu.xlsx) has been provided by the Account Manager as an example of the expected CPMU calculations
+- CPMU should be displayed to 5 decimal places
```
Comment (MattHodges-S4RB) [inline README.md]: 5 seems like a lot, if going with it the example table below should probably also round to 5

Comment (adamwitko-s4rb) [reply README.md]: It's on purpose that it says 5 and only displays 3 in the tables

Comment (MattHodges-S4RB) [reply README.md]: 👀

File: README.md [hunk 4]
```diff
@@ -1,20 +1,41 @@
...
 
-## 2. Update the route to additionally calculate "CPMU"
+## What we're looking for
 
-- CPMU = Complaints per million units
-- The calculation should happen server-side
-- Add any tests, if necessary
+- A working solution
+- Simplicity
+- Test Coverage
```
Comment (maciejkulinski) [inline README.md]: Test coverage

File: README.md [hunk 5]
```diff
@@ -1,20 +1,41 @@
-# Interview Questions #
+# Task
 
-Imagine you have been tasked by the Account Manager to create a basic reporting application that will allow the account manager to view complaints via a online portal.
-The company has provided you with a CSV file with all the complaints that the business has recieved and the number of corresponding sales in that period.
+You have been tasked by the Account Manager to create a reporting application that will allow the Account Manager to view complaints data via an online portal.
 
-## 1. Create an express app with a GET /data/cpmu route
+The company has provided you with a CSV file with all the complaints that the business has received and the number of corresponding sales in that period.
```
Comment (maciejkulinski) [inline README.md]: ...in the **given** period...

?

File: README.md [hunk 6]
```diff
@@ -1,20 +1,41 @@
...
 
-- The route or application should read the data from the CSV file and convert it to JSON.
-- Your solution should include a test for the correct conversion of the data to JSON.
+Your solution will be incorporated into the existing reporting product if it adds the value the Account Manager believes it will. Keep this in mind.
```
Comment (maciejkulinski) [inline README.md]: What are we trying to achieve here? Why mentioning Account Manager?

File: README.md [hunk 7]
```diff
@@ -1,20 +1,41 @@
...
-- The calculation should happen server-side
-- Add any tests, if necessary
+- A working solution
+- Simplicity
+- Test Coverage
+- A reasonable structure to your solution
+- Clear direction on how to make modifications to your solution
 
-For example:
+## Your Submission
```
Comment (maciejkulinski) [inline README.md]: You submission

File: README.md [hunk 8]
```diff
@@ -1,20 +1,41 @@
...
+
+- Pushed to a Github repository
+- Shared with your Technical contact at S4RB
```
Comment (maciejkulinski) [inline README.md]: technical contact

File: README.md [hunk 9]
```diff
@@ -1,20 +1,41 @@
...
+- If you're invited for a chat with S4RB we'll aim to talk about your submission. Be ready for questions on your solution!
+
+## 1. Serve up the [complaints](/data/complaints.csv) data from an API endpoint
```
Comment (maciejkulinski) [inline README.md]: Feels like this should this be on a different level than "What we're looking for" and "Your submission". Above is more like general guidelines, this is specific.

**Comment by adamwitko-s4rb** [comment]
@maciejkulinski mind having a look as I want to get this sent out to a candidate. Cheers.

**Comment by maciejkulinski** [comment]
> @maciejkulinski mind having a look as I want to get this sent out to a candidate. Cheers.

On it.

**Pull request merged**